error seemingly from `pythran`:

```
[701/1301] Compiling C++ object scipy/linalg/_linalg_pythran.cp313-win_amd64.pyd.p/meson-generated__linalg_pythran.cpp.obj
FAILED: [code=2] scipy/linalg/_linalg_pythran.cp313-win_amd64.pyd.p/meson-generated__linalg_pythran.cpp.obj 
"cl.exe" "-Iscipy\linalg\_linalg_pythran.cp313-win_amd64.pyd.p" "-Iscipy\linalg" "-I..\scipy\linalg" "-I..\.github\workflows\.pixi\envs\scipy-openblas\Lib\site-packages\pythran" "-I..\.github\workflows\.pixi\envs\scipy-openblas\Lib\site-packages\numpy\_core\include" "-ID:\a\scipy\scipy\.github\workflows\.pixi\envs\scipy-openblas\Include" "/MD" "/nologo" "/showIncludes" "/utf-8" "/Zc:__cplusplus" "/W2" "/EHsc" "/std:c++17" "/permissive-" "/O2" "/Zi" "/bigobj" "-DNPY_NO_DEPRECATED_API=NPY_1_9_API_VERSION" "-DENABLE_PYTHON_MODULE" "-D__PYTHRAN__=3" "-DPYTHRAN_BLAS_NONE" "/Fdscipy\linalg\_linalg_pythran.cp313-win_amd64.pyd.p\meson-generated__linalg_pythran.cpp.pdb" /Foscipy/linalg/_linalg_pythran.cp313-win_amd64.pyd.p/meson-generated__linalg_pythran.cpp.obj "/c" scipy/linalg/_linalg_pythran.cp313-win_amd64.pyd.p/_linalg_pythran.cpp
D:\a\scipy\scipy\.github\workflows\.pixi\envs\scipy-openblas\Lib\site-packages\pythran\pythonic/include/types/ndarray.hpp(484): error C2559: 'std::enable_if<`anonymous-namespace'::pythonic::types::is_numexpr_arg<F>::value&&std::is_same<bool,F::dtype>::value&&F::value==1&&!`anonymous-namespace'::pythonic::types::is_pod_array<F>::value,`anonymous-namespace'::pythonic::types::numpy_vexpr<`anonymous-namespace'::pythonic::types::ndarray<T,pS>,`anonymous-namespace'::pythonic::types::ndarray<long,`anonymous-namespace'::pythonic::types::pshape<long>>>>::type `anonymous-namespace'::pythonic::types::ndarray<T,pS>::operator [](const F &) const': cannot overload a member function without ref-qualifier with a member function with ref-qualifier
```

--- a/.github/workflows/pixi.toml
+++ b/.github/workflows/pixi.toml
@@ -163,7 +163,7 @@ env = { CC = "ccache $CC", CXX = "ccache $CXX", FC = "ccache $FC" }
 description = "Build SciPy (default settings)"
 
 [feature.build-task.target.win-64.tasks.build]
-cmd = "spin build --setup-args=-Dblas=blas --setup-args=-Dlapack=lapack --setup-args=-Duse-g77-abi=true  && cp .github/workflows/.pixi/envs/build/Library/bin/openblas.dll build-install/Lib/site-packages/scipy/linalg/openblas.dll"
+cmd = "spin build --setup-args=-Dblas=blas --setup-args=-Dlapack=lapack --setup-args=-Duse-g77-abi=true && cp .github/workflows/.pixi/envs/build/Library/bin/openblas.dll build-install/Lib/site-packages/scipy/linalg/openblas.dll"
 cwd = "../.."
 env = { CC = "ccache clang-cl", CXX = "ccache clang-cl", FC = "ccache $FC", FC_LD = "lld-link" }
 description = "Build SciPy (default settings)"
@@ -314,7 +314,7 @@ env = { CC = "ccache $CC", CXX = "ccache $CXX", FC = "ccache $FC" }
 description = "Build SciPy with scipy-openblas32"
 
 [feature.scipy-openblas.target.win-64.tasks.build-scipy-openblas]
-cmd = "spin build --build-dir=build-scipy-openblas --with-scipy-openblas && cp .github/workflows/.pixi/envs/scipy-openblas/Library/bin/openblas.dll build-scipy-openblas/Lib/site-packages/scipy/linalg/openblas.dll"
+cmd = "spin build --build-dir=build-scipy-openblas --with-scipy-openblas && cp .github/workflows/.pixi/envs/scipy-openblas/Library/bin/openblas.dll build-scipy-openblas-install/Lib/site-packages/scipy/linalg/openblas.dll"
 cwd = "../.."
 env = { CC = "ccache clang-cl", CXX = "ccache clang-cl", FC = "ccache $FC", FC_LD = "lld-link" }
 description = "Build SciPy with scipy-openblas32"

--- a/.github/workflows/pixi.toml
+++ b/.github/workflows/pixi.toml
@@ -313,12 +313,6 @@ cwd = "../.."
 env = { CC = "ccache $CC", CXX = "ccache $CXX", FC = "ccache $FC" }
 description = "Build SciPy with scipy-openblas32"
 
-[feature.scipy-openblas.target.win-64.tasks.build-scipy-openblas]
-cmd = "spin build --build-dir=build-scipy-openblas --with-scipy-openblas && cp .github/workflows/.pixi/envs/scipy-openblas/Library/bin/openblas.dll build-scipy-openblas-install/Lib/site-packages/scipy/linalg/openblas.dll"
-cwd = "../.."
-env = { CC = "ccache clang-cl", CXX = "ccache clang-cl", FC = "ccache $FC", FC_LD = "lld-link" }
-description = "Build SciPy with scipy-openblas32"
-
 [feature.scipy-openblas.tasks.test-scipy-openblas]
 cmd = "spin test --no-build --build-dir=build-scipy-openblas"
 cwd = "../.."

--- a/.github/workflows/pixi.toml
+++ b/.github/workflows/pixi.toml
@@ -162,6 +162,12 @@ cwd = "../.."
 env = { CC = "ccache $CC", CXX = "ccache $CXX", FC = "ccache $FC" }
 description = "Build SciPy (default settings)"
 
+[feature.build-task.target.win-64.tasks.build]
+cmd = "spin build --setup-args=-Dblas=blas --setup-args=-Dlapack=lapack --setup-args=-Duse-g77-abi=true  && cp .github/workflows/.pixi/envs/build/Library/bin/openblas.dll build-install/Lib/site-packages/scipy/linalg/openblas.dll"
+cwd = "../.."
+env = { CC = "ccache clang-cl", CXX = "ccache clang-cl", FC = "ccache $FC", FC_LD = "lld-link" }
+description = "Build SciPy (default settings)"
+
 
 ### Default testing ###
 
@@ -305,6 +311,12 @@ scipy-openblas32 = "*"
 cmd = "spin build --build-dir=build-scipy-openblas --with-scipy-openblas"
 cwd = "../.."
 env = { CC = "ccache $CC", CXX = "ccache $CXX", FC = "ccache $FC" }
+description = "Build SciPy with scipy-openblas32"
+
+[feature.scipy-openblas.target.win-64.tasks.build-scipy-openblas]
+cmd = "spin build --build-dir=build-scipy-openblas --with-scipy-openblas && cp .github/workflows/.pixi/envs/scipy-openblas/Library/bin/openblas.dll build-scipy-openblas/Lib/site-packages/scipy/linalg/openblas.dll"
+cwd = "../.."
+env = { CC = "ccache clang-cl", CXX = "ccache clang-cl", FC = "ccache $FC", FC_LD = "lld-link" }
 description = "Build SciPy with scipy-openblas32"
 
 [feature.scipy-openblas.tasks.test-scipy-openblas]

--- a/.github/workflows/windows.yml
+++ b/.github/workflows/windows.yml
@@ -44,14 +44,14 @@ jobs:
 
       - name: Build
         working-directory: .github/workflows
-        run: pixi run build-scipy-openblas
+        run: pixi run build
 
       - name: Test
         working-directory: .github/workflows
         run: |
           # test runner parallel clashes with OpenBLAS multithreading
           $env:OPENBLAS_NUM_THREADS=1
-          pixi run test-scipy-openblas -j4 -- --durations=25
+          pixi run test -j4 -- --durations=25
 
   #############################################################################
   full_spin_min_numpy:

